Modules/_ctypes/malloc_closure.c

```diff
@@ -89,16 +89,27 @@ static void more_core(void)
 /******************************************************************/
 
 /* put the item back into the free list */
-void ffi_closure_free(void *p)
+void Py_ffi_closure_free(void *p)
 {
+#if USING_APPLE_OS_LIBFFI
+    if (__builtin_available(macos 11, ios 13, watchos 6, tvos 13, *)) {
```

Comment:
`__builtin_available` again

Reply:
I'm afraid this needs a configure test, ffi_closure_free is not available at all in older SDKs.